lib/localdb.js

```diff
@@ -30,7 +30,8 @@ function getDitchHandle(cb) {
       my_db_config = parseMongoConnectionURL(mongoConnectionURL);
     }
     catch (e) {
-      var errMessage = new Error("Incorrect format for database connection string.");
+      my_db_logger.error(e)
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)